src/commcare_cloud/commands/terraform/aws.py

```diff
@@ -94,11 +95,37 @@ def run(self, args, unknown_args):
 
 class AwsFillInventory(CommandBase):
     command = 'aws-fill-inventory'
-    help = "Fill inventory.ini.j2 using existing AWS resources"
+    help = """
+        Fill inventory.ini.j2 using AWS resource values cached in aws-resources.yml
+
+        If --cached is not specified, also refresh aws-resources.yml
+        to match what is actually in AWS.
+    """
+
+    arguments = [
+        Argument('--cached', action='store_true', help="""
+            Use the values set in aws-resources.yml rather than fetching from AWS.
+
+            This runs much more quickly and gives the same result, provided no changes
+            have been made to our actual resources in AWS.
+        """)
+    ]
 
     def run(self, args, unknown_args):
         environment = get_environment(args.env_name)
-        resources = get_aws_resources(environment)
+        if not os.path.exists(environment.paths.inventory_ini_j2):
+            print("Env {} not using templated inventory (inventory.ini.j2). Skipping"
```

Comment:
If there isn't an inventory j2 file, it seems like it would need to be added back for this to work, should the print statement be updated to reflect that?

Reply:
Most of our environments don't use templated inventory files, only the ones using AWS, which is staging and production. I added this here to make it easier to add the travis test which runs `commcare-cloud ${env} aws-fill-inventory --cached` for each environment; I don't expect people to try to run it for other environments, but if they do, I think this message has enough information to orient them, and if it comes up often in a situation where more information could be helpful, happy to add more then.